src/components/generics/vacation-test-forms/employee-vacation-requests.tsx

```diff
@@ -551,14 +647,15 @@ const RenderEmployeeVacationRequests = ({ persons }: Props) => {
               </TableRow>
             </TableHead>
             <TableBody>
-              {sortedVacationRequests.map((request: VacationRequest, index: number) => (
+              {!loading && sortedVacationRequests.map((request: VacationRequest, index: number) => (
                 <>
                   <StyledTableRow key={ request.id }>
-                    <StyledTableCell component="th" scope="row">{ handleRequestType(request.type)}</StyledTableCell>
-                    <StyledTableCell>{ handlePersonNames(Number(request.personId)) }</StyledTableCell>
+                    <StyledTableCell component="th" scope="row">{ getLocalizedRequestType(request.type)}</StyledTableCell>
+                    <StyledTableCell>{ request.personId ? handlePersonNames(request.personId) : handlePersonNames(null) }</StyledTableCell>
```

Comment:
There is no need to conditionally call this function in this way as the function will handle request.personId if it is null

Reply:
True. Made the call not to check for personId. Had to change "handlePersonNames" props from (id: string | null) to (id: string | undefined) though.

```suggestion
                    >
```